.github/workflows/kairos-ubuntu-22-lts.yml

```diff
@@ -0,0 +1,48 @@
+---
+name: Build Kairos Ubuntu 22 LTS
+
+on: # yamllint disable-line rule:truthy
+  push:
+    branches:
+      - main
+    paths:
+      - images/kairos-ubuntu-22-lts/**
+  pull_request:
+    branches:
+      - main
+    paths:
+      - images/kairos-ubuntu-22-lts/**
+
+jobs:
+  build-container:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - uses: earthly/actions-setup@v1.0.8
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          version: "0.7.1"
+      - uses: actions/checkout@v4
+      - uses: docker/setup-qemu-action@v2
```

Comment:
Is this needed? I thought we could use the USB image build workflow as a base. Same with the docker login.

Reply:
qemu is needed in order to build for arm